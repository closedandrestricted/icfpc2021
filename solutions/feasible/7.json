{"vertices":[[21,36],[14,34],[18,30],[11,30],[11,40],[14,34],[17,28],[24,27],[18,29],[20,35],[13,33],[20,30],[16,35],[18,28],[25,30],[22,42],[20,30],[18,36],[11,40],[14,33],[16,40],[14,33],[19,29],[16,35],[18,39],[24,37],[12,33],[11,40],[14,34],[17,27],[22,32],[18,41],[16,39],[13,33],[13,40],[16,34],[25,38],[24,45],[14,33],[13,40],[13,33],[17,27],[21,33],[20,37],[14,33],[20,35],[13,33],[19,39],[12,39],[11,27],[14,33],[18,39],[23,34],[19,42],[11,39],[14,33],[24,28],[22,35]]}