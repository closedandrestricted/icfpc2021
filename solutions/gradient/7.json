{"vertices":[[25,63],[24,56],[16,39],[16,32],[32,75],[29,68],[23,49],[21,43],[11,27],[18,74],[25,72],[27,61],[29,54],[21,37],[14,34],[24,70],[30,66],[34,60],[30,47],[27,40],[29,65],[34,60],[36,53],[37,46],[28,33],[35,34],[33,58],[38,53],[40,46],[39,38],[36,31],[38,27],[38,53],[45,51],[44,40],[45,33],[42,27],[39,20],[45,52],[51,46],[51,39],[49,27],[45,21],[51,48],[58,49],[58,39],[55,32],[47,14],[44,7],[65,52],[62,45],[55,25],[53,18],[51,7],[60,39],[58,32],[55,11],[58,4]]}