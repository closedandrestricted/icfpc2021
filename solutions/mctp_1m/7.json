{"vertices":[[45,27],[42,33],[44,36],[44,29],[44,27],[39,23],[36,29],[37,36],[37,30],[52,23],[51,30],[39,17],[41,24],[36,29],[30,32],[59,22],[44,28],[38,24],[46,29],[41,34],[53,25],[50,31],[39,17],[43,22],[34,36],[40,34],[48,31],[48,23],[41,24],[43,30],[36,28],[36,41],[48,25],[55,22],[45,30],[49,35],[31,33],[29,39],[42,27],[52,28],[50,35],[44,40],[37,37],[45,33],[52,35],[43,37],[37,41],[30,39],[23,36],[47,40],[41,43],[36,48],[29,45],[16,39],[46,48],[42,53],[24,50],[19,46]]}