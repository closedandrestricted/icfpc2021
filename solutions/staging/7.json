{"vertices":[[15,30],[20,35],[20,55],[18,62],[28,33],[21,34],[25,41],[22,48],[13,68],[36,31],[33,38],[26,38],[32,41],[20,55],[16,61],[43,33],[36,32],[33,39],[29,48],[26,54],[48,28],[43,33],[29,34],[31,41],[33,56],[36,63],[46,21],[39,26],[37,32],[39,43],[37,50],[31,56],[47,14],[44,21],[43,35],[46,41],[44,52],[37,53],[51,8],[50,24],[49,31],[48,48],[42,46],[44,10],[47,17],[54,36],[52,42],[49,47],[56,48],[52,22],[53,29],[53,35],[49,40],[63,51],[54,35],[53,42],[56,39],[57,46]]}