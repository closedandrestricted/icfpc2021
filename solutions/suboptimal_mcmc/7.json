{"vertices":[[40,17],[39,10],[55,14],[59,8],[31,27],[38,24],[42,17],[49,18],[54,14],[17,28],[24,27],[44,28],[39,24],[47,25],[51,20],[11,27],[31,27],[38,27],[42,31],[49,32],[18,29],[25,31],[35,33],[39,38],[47,39],[50,45],[18,36],[29,37],[35,40],[36,46],[43,46],[53,53],[24,39],[28,44],[36,46],[33,53],[39,52],[46,51],[31,40],[34,48],[38,53],[28,59],[34,57],[32,47],[31,54],[35,59],[33,65],[30,63],[23,66],[37,59],[33,65],[33,72],[26,69],[19,72],[35,72],[38,78],[20,73],[14,77]]}