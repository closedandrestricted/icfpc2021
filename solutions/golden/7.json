{"vertices":[[18,63],[19,56],[23,36],[17,32],[16,76],[19,69],[23,50],[25,43],[11,27],[30,75],[23,73],[23,63],[26,56],[22,37],[16,33],[37,75],[26,66],[29,59],[29,49],[27,42],[35,68],[32,61],[34,54],[36,47],[32,38],[31,31],[36,60],[37,54],[38,47],[38,39],[37,32],[38,27],[38,53],[42,48],[42,41],[44,34],[41,26],[43,32],[44,52],[49,47],[49,40],[47,27],[45,20],[49,47],[56,48],[56,38],[53,31],[47,13],[44,6],[63,51],[60,44],[56,24],[53,17],[51,6],[56,38],[58,31],[55,10],[58,3]]}